modules/rostests/apitests/shell32/GetDisplayNameOf.cpp

```diff
@@ -0,0 +1,40 @@
+/*
+ * PROJECT:         ReactOS API tests
+ * LICENSE:         LGPLv2.1+ - See COPYING.LIB in the top level directory
+ * PURPOSE:         Test for GetDisplayNameOf
+ * COPYRIGHT:       Copyright 2023 Mark Jansen <mark.jansen@reactos.org>
```

Comment:
```suggestion
 * COPYRIGHT:   Copyright 2023 Mark Jansen <mark.jansen@reactos.org>
```

Reply:
Why spam a suggestion per line?
Just suggest the entire block in one go...